var/spack/repos/builtin/packages/hdf5-vol-external-passthrough/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Hdf5VolExternalPassthrough(CMakePackage):
+    """Package for HDF5 external pass-through VOL."""
+
+    homepage = "https://sdm.lbl.gov/"
+    git      = "https://github.com/hpc-io/vol-external-passthrough.git"
+    maintainers = ['hyoklee']
+
+    version('v1.0')
```

Comment:
A version outside of `git` with a checksum-able tarball is also prefereable:
```python
url      = "https://github.com/hpc-io/vol-external-passthrough/archive/refs/tags/v1.0.tar.gz"
git      = "https://github.com/hpc-io/vol-external-passthrough.git"

version('develop', branch='develop')
version('1.0', sha256='99a06d1c31451f8f0c8c10fec112410cda1f951f0eda1bd0ca999d6b35cf7740')
```

Reply:
Thanks!  Testing.  I'll also test if replacing line 17 with "depends_on('hdf5@1.13.0:')" will spec and install with the currently available 'hfd5@develop-1.13', because it does with the hdf5-vol-log.  hdf5@1.13.0 will be the correct minimum version and will be available within a couple of weeks.  If it works I'll add that change, too.

Reply:
Done